booklore/docker-compose.yml

```
@@ -0,0 +1,32 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: booklore_web_1
+      APP_PORT: 6060
+      
+  web:
+    image: booklore/booklore:v1.2.1@sha256:f0169c2a65859650860f72f8c3f9211c3260a4fe99f1e9fd2f94555f96a02fa0
+    restart: on-failure
+    environment:
+      DATABASE_URL: jdbc:mariadb://db:3306/booklore
```

Use full service names.
```suggestion
      DATABASE_URL: jdbc:mariadb://booklore_db_1:3306/booklore
```